File: frontend/src/components/ShowQuestion/Answer.js
import Vote from "./vote";
const Answer = ({answer}) => {
    return (
        <div className="answer">
            <h3>{answer.poster}</h3>
            {answer.score}
            <Vote answerId={answer.id}/>
            <p>{answer.body}</p>
        </div>
    )
}
export default Answer;

File: app/models/answer.rb
class Answer < ApplicationRecord
    validates :poster_id, :question_id, :body, presence: false

    belongs_to :poster,
    primary_key: :id,
    foreign_key: :poster_id,
    class_name: :User

    belongs_to :question,
    primary_key: :id,
    foreign_key: :question_id,
    class_name: :Question

    has_many :votes,
    primary_key: :id,
    foreign_key: :question_id,
    class_name: :Vote
end
